With Plugin Redirect: fix i18n string

retrofox opened the pull request:
#### Changes proposed in this Pull Request

This PR updates the i18n string defined by the plugin, replacing `'Redirecting to setup %(pluginName)s in five seconds.'` by `'Redirecting to setup for %(pluginName)s in five seconds.'`

#### Testing instructions

Check the i18n string is correct.

retrofox committed:
with-plugin-redirect: fix 1i8n string

a8ci18n commented on client/my-sites/plugins/with-plugin-redirect/index.js:
```diff
@@ -157,11 +157,14 @@ const withPluginRedirect = createHigherOrderComponent(
-					},
-				} );
+				const redirectMessage = translate(
+					'Redirecting to setup for %(pluginName)s in five seconds.',
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'Redirecting to setup %(pluginName)s in five seconds.' )` *ES Score: 9*

pablinos commented:
I don't want to get too pedantic about grammar, as it's definitely not my strength, but I think this should either be:

> Redirecting to the setup for %(pluginName)s in five seconds.

or

> Redirecting to set up %(pluginName)s in five seconds.

Currently we're using 'setup' as a verb, which is wrong, it should be two words. Adding 'for' suggests that 'setup' is a noun, so I think it needs 'the' added.

But, as I say, I'm not an expert in these things.

retrofox commented:
> I don't want to get too pedantic about grammar, as it's definitely not my strength, but I think this should either be:
> 
> > Redirecting to the setup for %(pluginName)s in five seconds.
> 
> or
> 
> > Redirecting to set up %(pluginName)s in five seconds.
> 
> Currently we're using 'setup' as a verb, which is wrong, it should be two words. Adding 'for' suggests that 'setup' is a noun, so I think it needs 'the' added.
> 
> But, as I say, I'm not an expert in these things.

added `the` to the string. rebased.

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5949388

Hi @retrofox, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Redirecting to the setup for %(pluginName)s in five seconds.`

Thank you in advance!

a8ci18n commented on client/my-sites/plugins/with-plugin-redirect/index.js:
```diff
@@ -157,11 +157,14 @@ const withPluginRedirect = createHigherOrderComponent(
-					},
-				} );
+				const redirectMessage = translate(
+					'Redirecting to the setup for %(pluginName)s in five seconds.',
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Redirecting to setup for %(pluginName)s in five seconds.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/with-plugin-redirect-i18n)

allilevine reviewed (approved):
Looks good ✨ 
<img width="603" alt="Screen Shot 2021-05-26 at 4 14 29 PM" src="https://user-images.githubusercontent.com/1689238/119725109-8a72e480-be3d-11eb-8d05-21615d50cb44.png">

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5949388).
